fhirResourcesToLoad/o2billy_04_device-request-rad-E0470.json

```diff
@@ -11,13 +11,13 @@
     {
```

Comment:
I know this is not this PR but the profile is not correct.
line 7 should be 'http://hl7.org/fhir/us/davinci-crd/StructureDefinition/profile-devicerequest-r4'

Reply:
I don't see US Core has DeviceRequest spec. Can you send your reference? Thanks.

Reply:
@yunwwang I am wondering if we should create another PR to fix those test data? It seems there are a lot of issues similar to this for multiple structured data.

Reply:
Not US Core profile. It is CRD profile.

Reply:
http://hl7.org/fhir/us/davinci-crd/2019May/profile-devicerequest-r4.html